examples_tests/31.SkinningDataBenchmark/main.cpp

```diff
@@ -39,7 +51,11 @@ int main()
     params.Bits = 24; //may have to set to 32bit for some platforms
     params.ZBufferBits = 24; //we'd like 32bit here
     params.DriverType = video::EDT_OPENGL; //! Only Well functioning driver, software renderer left for sake of 2D image drawing
+#ifndef BENCHMARK
     params.WindowSize = dimension2d<uint32_t>(1280, 720);
+#else
+    params.WindowSize = dimension2d<uint32_t>(64, 64);
```

Comment:
for benchmark mode you want to make a depth only FBO and render to that, not the screen.